src/extension/api.ts

```diff
@@ -126,7 +130,31 @@ export async function fetchEmbedding(request: StreamRequest) {
 
   } catch (error: unknown) {
     if (error instanceof Error) {
-      console.error("Fetch error:", error)
+      logConsoleError(ErrorType.Fetch_Error, 'Fetch error', error);
+      fitchErrorDetection(error)
     }
   }
 }
+
+//Define an array containing all the error messages that need to be detected when fetch error occurred
+const knownErrorMessages = [
+  "First parameter has member 'readable' that is not a ReadableStream.", //This error occurs When plugins such as Fitten Code are enabled
+  "The 'transform.readable' property must be an instance of ReadableStream. Received an instance of h" //When you try to enable the Node.js compatibility mode Compat to solve the problem, this error may pop up
+];
+
+function fitchErrorDetection(error: Error) {
```

Comment:
This can be moved in to `extension/utils.ts` and I think maybe we could rename to `notifyKnownErrors`.